test/integration/servicecluster.go

```diff
@@ -194,6 +194,13 @@ func newServiceClusterSuite(
 
 		t.Log("CatalogEntry & CustomResourceDiscoverySet exists")
 
+		// Check the Catapult dynamic webhook service is deployed.
+		webhookService := &corev1.Service{}
```

Comment:
This catalog update is orthogonal to the elevator PR it's currenly in. Please move it to separate PR and have it quick lgtmed

Reply:
moved to #292